settings.gradle.kts

```diff
@@ -3,3 +3,29 @@ rootProject.name = "rep-bot"
 plugins {
     id("org.gradle.toolchains.foojay-resolver-convention") version("0.6.0")
 }
+
+dependencyResolutionManagement {
+    versionCatalogs {
+        create("libs") {
+            // misc
+            library("jetbrains-annotations", "org.jetbrains:annotations:24.0.1")
+            version("sadu", "1.3.1-DEV")
```

Comment:
We are going to a dev version again?

Reply:
Actually I think we dont need to... We dont use the feature which is fixed there.